reana_client/cli/workflow.py

```diff
@@ -129,7 +129,8 @@ def workflow_workflows(ctx, sessions, _filter, output_format, access_token,
         response = get_workflows(access_token, type, bool(verbose), block_size)
         verbose_headers = ['id', 'user', 'size']
         headers = {
-            'batch': ['name', 'run_number', 'created', 'status'],
+            'batch': ['name', 'run_number', 'created', 'run_started_at',
```

Comment:
I think @tiborsimko wanted to call these columns `started` and `completed` see - https://github.com/reanahub/reana-client/issues/370#issue-561578866

Reply:
Can you bump up the devYYYYMMDD version number as part of the commit?